src/modules/content-tree/content.tree.module.js

```diff
@@ -43,6 +44,8 @@ export default class ContentTreeModule extends Component {
         loadSubtree(this.props.restInfo, this.subtree, (loadedSubtree) => {
             this.setInitialItemsState(loadedSubtree[0]);
         });
+
+        document.body.addEventListener('ez-content-tree-refresh', this.refreshContentTree, false);
```

Comment:
This `componentDidMount()` has the early return. There is a possibility this event will not be attached. Seems like should be before the return or am I missing something?

Reply:
You're right, good catch. :)